.changeset/rename-dynamic-workflows-client-js.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/client-js': minor
+---
+
+Renamed the stored workflow client methods to dynamic workflows: `upsertStoredWorkflow()`, `listStoredWorkflows()`, and `getStoredWorkflow()` are now `upsertDynamicWorkflow()`, `listDynamicWorkflows()`, and `getDynamicWorkflow()`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)